src/package.json

```diff
@@ -9,7 +9,7 @@
     "mocha": "mocha --full-trace tests/unit/",
     "test": "npm run lint && mocha tests/unit/",
```

Comment:
General comment: I am not sure test should contain the lint check as well. maybe we should have an npm build or something similar?